frame/nomination-pools/src/lib.rs

```diff
@@ -2470,12 +2470,17 @@ impl<T: Config> Pallet<T> {
 
 		for id in reward_pools {
 			let account = Self::create_reward_account(id);
-			assert!(
```

Comment:
Why did you removed the assert?

Reply:
I did, it does not hold anymore because we changed ED in Kusama :(

Reply:
Context https://github.com/paritytech/polkadot/pull/6372#discussion_r1038971348